include/rmm/cuda_stream_view.hpp

```diff
@@ -58,16 +70,22 @@ class cuda_stream_view {
 
   /**
    * @brief Implicit conversion to cudaStream_t.
+   *
+   * @return cudaStream_t The wrapped stream.
    */
   constexpr operator cudaStream_t() const noexcept { return value(); }
 
   /**
    * @brief Return true if the wrapped stream is the CUDA per-thread default stream.
+   *
+   * @return true if the wrapped stream is the CUDA per-thread default stream.
    */
   [[nodiscard]] inline bool is_per_thread_default() const noexcept;
 
   /**
-   * @brief Return true if the wrapped stream is explicitly the CUDA legacy default stream.
+   * @brief Check if the wrapped stream is explicitly the CUDA legacy default stream.
+   *
+   * @return true if the wrapped stream is the CUDA legacy default stream.
```

Comment:
Can we not duplicate?

```suggestion
   * @brief Check if the referenced stream is explicitly the CUDA legacy default stream.
   *
   * @return true if the referenced stream is the CUDA legacy default stream.
```